mmv1/third_party/terraform/.teamcity/tests/nightly_tests_project.kt

```diff
@@ -36,6 +36,13 @@ class NightlyTestProjectsTests {
                 }
             }
             assertTrue("Build configuration `${bt.name}` contains a CRON trigger", found)
+
+            // Check that nightly test is being ran on main branch
+            var isDefault: Boolean = false
+            if (bt.triggers.vcs.branchFilter == "+:ref/head/main"){
```

Comment:
I think there may be an import problem here, from this error:

```
Error:  Error while generating TeamCity configs:
Error: [ERROR] Compilation error tests/nightly_tests_project.kt[42:29]: Unresolved reference: vcs
```

Reply:
This took a bit to figure out. Went ahead and added a `lateinit var schedulingTrigger: ScheduleTrigger` variable that would be initialized later on when the trigger type is found to be `schedulingTrigger`

From there the `schedulingTrigger` would be initialized with `item as SchedulingTrigger`. Since we set the trigger to be a Scheduling trigger we can get the branch filter with `schedulingTrigger.branchFilter`